assests/javaScript/home.js

```diff
@@ -0,0 +1,49 @@
+const movieList = document.getElementById("movie-list");
+const apiKey = "d2aa1d6a7ef9316cd7e2270dd937b843";
+
+async function fetchMovieList() {
+  try {
+    const fetchedData = await fetch(
+      `https://api.themoviedb.org/3/movie/top_rated?api_key=${apiKey}&language=en-US&page=1`
+    );
+
+    const movieList = await fetchedData.json();
+
+    showMovies(movieList.results);
+  } catch (erorr) {
+    showErrorMessage();
+  }
+}
```

Comment:
Nicely done :+1:
Just a suggestions, when you don't want to use `error` object inside a catch block you can just remove it.

Reply:
Thanksssss. yes i delete it.